garak/translators/remote.py

```diff
@@ -105,6 +122,10 @@ class DeeplTranslator(Translator):
         "en"
     ]
     # fmt: on
+    # Applied when a service only supports regions specific codes
+    bcp47_overrides = {
+        "en": "en-US",
```

Comment:
I'm disdainful of not making English English default English. I guess empire instincts die hard.